### PR TITLE
Update plugin width and height defaults

### PR DESCRIPTION
**Pull request by jvega190**

### CODE REVIEW

**Review by sumerjabri — APPROVED** [review]

**Review by sumerjabri — CHANGES_REQUESTED** [review]
@jvega190 please update the version number to make it easier to publish the plugin.

**Comment by jvega190** [comment]
@sumerjabri done, version updated

**Review by rart — APPROVED** [review]

**Review by sumerjabri — APPROVED** [review]

**Comment by jvega190** [comment]
@rart @sumerjabri I sent an update to support z+click to follow link when no API key is set.

**Review by sumerjabri — APPROVED** [review]

File: delivery/templates/googlemaps.ftl [hunk 1]
```diff
@@ -33,21 +33,23 @@
...
-      src="https://www.google.com/maps/embed/v1/place?key=${apiKey}&${params?join('&')}"
-    />
+    <div class="craftercms-googlemaps-plugin-container">
+      <@crafter.iframe
+        allowfullscreen="true"
+        width="${contentModel.width_s}"
+        height="${contentModel.height_s}"
+        frameborder="0" style="border:0"
+        src="https://www.google.com/maps/embed/v1/place?key=${apiKey}&${params?join('&')}"
+      />
```
Comment (rart) [inline delivery/templates/googlemaps.ftl]: Should either have a field or not be a `@crafter` element

**Review by rart — CHANGES_REQUESTED** [review]

File: delivery/templates/googlemaps.ftl (hunk 1 above)
Comment (jvega190) [reply delivery/templates/googlemaps.ftl]: Correct, it doesn't represent a field in the component, changing it to be an iframe element

**Review by rart — APPROVED** [review]

**Pull request merged**